css/style.css

```diff
@@ -689,6 +699,42 @@ ul.chars{
     margin: 0 auto;
 }
 
+@media only screen and (max-device-width:1742px) {
+    .page-wide {
+        width: 100%;
+        margin: 0 auto;
+    }
+    a.nav {
+        text-align: center;
+        font-size:0.5em
+    }
+    .nav-pages {
+        float: none;
+    }
+    .brand-logo {
+        display: none;
+    }
+    .brand-link {
+        float: none;
+    }
+    .home-text {
+        display: block !important;
+    }
+    .home-main {
+        width: 100%;
+    }
+    .sidebar {
+        width: 100% !important;
+    }
+    .title-header-text {
+        font-size: 15vw;
```

Comment:
perhaps the one acceptable use of viewport units, great for logotype-style-text but still doesn’t compose well

Reply:
this will be all replaced by hamburger menu, no? i prefer just redoing that here instead of trying to format the weird stuff we have rn